conf/cmi/block.block.chatleijuke.yml

```diff
@@ -7,17 +7,19 @@ dependencies:
     - system
   theme:
     - hdbt
-id: watsonchatbot
+id: chatleijuke
 theme: hdbt
 region: attachments
 weight: 0
 provider: null
-plugin: watson_chatbot
+plugin: chat_leijuke
 settings:
-  id: watson_chatbot
-  label: 'Watson chatbot'
+  id: chat_leijuke
+  label: 'Chat Leijuke Watson'
   label_display: '0'
   provider: helfi_platform_config
+  chat_title: Asunnonhakubotti
```

Comment:
I assume we need proper English translation for this and FI & SV translations as well.